other/upgrade_2-1_mysql.sql

```diff
@@ -2547,11 +2547,16 @@ ALTER TABLE {$db_prefix}log_actions
 ADD INDEX id_topic_id_log (id_topic, id_log);
 ---#
 
----# Updating log_activity
+---# Updating log_activity mostOn
```

Comment:
typo :/

Reply:
What is the typo?   

Oddly, over the years, there have been multiple spellings of this index.   Hopefully this cleans up all of them.

Reply:
@Antes ?   What is the typo - please be specific?

Reply:
damn my bad actually I read O (capital o) as 0...

Reply:
Thx,